x/logger/logger.go

```diff
@@ -136,7 +136,7 @@ func (h *Handler) OnMeasurement(m modelx.Measurement) {
 			m.HTTPResponseStart.TransactionID,
 		)
 	}
-	if m.HTTPRoundTripDone != nil {
+	if m.HTTPRoundTripDone != nil && m.HTTPRoundTripDone.Error == nil {
```

Comment:
This is a fix to avoid printing unnecessary `> 0\n>` when there is, e.g., a TLS handshake error